src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/CalledLegacySegmentCollection.java

```diff
@@ -0,0 +1,107 @@
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Represents a CBS-style segmentation to enable IGV-compatible plotting.
+ *
+ * IGV ignores column headers and requires that no other headers are present.
+ * We use the conventional CBS-style column headers (which includes the sample name)
+ * and suppress the SAM-style metadata header (which breaks the contract for construction from input files).
+ * See <a href="http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide#CBS">
+ *     http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide#CBS</a>
+ * and <a href="https://software.broadinstitute.org/software/igv/SEG">
+ *     https://software.broadinstitute.org/software/igv/SEG</a>.
+ *
+ */
+public final class CalledLegacySegmentCollection extends AbstractSampleLocatableCollection<CalledLegacySegment> {
+    //note to developers: repeat the column headers in Javadoc so that they are viewable when linked
+    /**
+     * Sample, Chromosome, Start, End, Num_Probes, Call, Segment_Mean
+     */
+    enum CalledLegacySegmentTableColumn {
+        SAMPLE("Sample"),
+        CHROMOSOME("Chromosome"),
+        START("Start"),
+        END("End"),
+        NUM_PROBES("Num_Probes"),
+        CALL("Call"),
+        SEGMENT_MEAN("Segment_Mean");
+
+        private final String columnName;
+
+        CalledLegacySegmentTableColumn(final String columnName) {
+            this.columnName = columnName;
+        }
+
+        static final TableColumnCollection COLUMNS = new TableColumnCollection(
+                Stream.of(values()).map(c -> c.columnName).toArray());
+    }
+
+    private static final Function<DataLine, CalledLegacySegment> LEGACY_SEGMENT_DATA_LINE_TO_RECORD_FUNCTION = dataLine -> {
+        final String sampleName = dataLine.get(CalledLegacySegmentTableColumn.SAMPLE.columnName);
+        final String contig = dataLine.get(CalledLegacySegmentTableColumn.CHROMOSOME.columnName);
+        final int start = dataLine.getInt(CalledLegacySegmentTableColumn.START.columnName);
+        final int end = dataLine.getInt(CalledLegacySegmentTableColumn.END.columnName);
+        final int numProbes = dataLine.getInt(CalledLegacySegmentTableColumn.NUM_PROBES.columnName);
+        final double segmentMean = dataLine.getDouble(CalledLegacySegmentTableColumn.SEGMENT_MEAN.columnName);
+        final String callOutputString = dataLine.get(CalledCopyRatioSegmentCollection.CalledCopyRatioSegmentTableColumn.CALL);
+        final CalledCopyRatioSegment.Call call = Arrays.stream(CalledCopyRatioSegment.Call.values())
+                .filter(c -> c.getOutputString().equals(callOutputString)).findFirst().orElse(null);;
```

Comment:
Extra semicolon.  When can the call ever be `null`?

Reply:
Changed it to throw an exception.

Done